hedera-node/src/main/java/com/hedera/services/txns/crypto/CryptoAdjustAllowanceTransitionLogic.java

```diff
@@ -292,4 +319,12 @@ private boolean exceedsAccountLimit(final Account ownerAccount) {
 		}
 		return newSerials;
 	}
+
+	private void validateAllowanceLimitsOn(final Account owner) {
+		final var limitExceeded = exceedsAccountLimit(owner);
+		if (limitExceeded) {
+			sideEffectsTracker.reset(); // this is not needed
```

Comment:
🛑 :warning: This is actually a P1 bug, since it **clears the transfer list of the fee payments**! (But those payments are still going to be charged.)

Reply:
Should have been removed. Had a comment as well.